VAMobile/src/components/Carousel/Carousel.test.tsx

```diff
@@ -1,31 +1,33 @@
-import 'react-native'
 import React from 'react'
-// Note: test renderer must be required after react-native.
-import 'jest-styled-components'
-import { ReactTestInstance, act } from 'react-test-renderer'
-
-import { context, render, RenderAPI, waitFor } from 'testUtils'
+import { context, render, screen } from 'testUtils'
 import Carousel from './Carousel'
 import { TextView } from '../index'
 
 context('Carousel', () => {
-  let component: RenderAPI
-  let testInstance: ReactTestInstance
-  let t = jest.fn(() => {})
+  let t = jest.fn(() => { })
 
   const TestComponent = () => {
     return <TextView>Test Component</TextView>
   }
 
-  beforeEach(async () => {
-    await waitFor(() => {
-      component = render(<Carousel screenList={[{ name: 'TestComponent', component: TestComponent }]} onCarouselEnd={() => {}} translation={t} />)
-    })
+  const screenList = [
+    {
+      name: 'TestComponent',
+      component: TestComponent,
+      a11yHints: {
+        skipHint: 'skip',
+        doneHint: 'done',
+      }
+    }
+  ]
 
-    testInstance = component.UNSAFE_root
+  beforeEach(async () => {
```

Comment:
this async can still be removed

Reply:
Darn! I thought I got them all. Removed.